natgeo/mortar
# [SYTO-1097] Mobile Galaxy S6, Video Modal  is not showing correcty

https://jira.natgeo.com/browse/SYTO-1097

####How to test
1. ``gulp serve``
2. Scroll down until the Video Promo card
3. Use a mobile emulator and see if the modal render properly.
